### DISCUSSION TITLE
any older linux distros work good on v86 thats not dsl?

### DISCUSSION THREAD

**Discussion by ReconEvacuateDEVX** [copy/v86#551] [Q&A] [answered]
dsl was long dead before v86 and im wondering if there is better distros that work as good as dsl on v86 :>

**Comment by copy** [comment, marked as answer]
[Tiny Core](http://tinycorelinux.net/) or [Slax](https://www.slax.org/)

**Reply by ReconEvacuateDEVX** [reply]
> [Tiny Core](http://tinycorelinux.net/) or [Slax](https://www.slax.org/)

Thanks Mate!

**Reply by hello-smile6** [reply]
@ReconEvacuateDEVX Ubuntu 10 also works.

**Reply by Darin755** [reply]
alpine linux works as well
